lib/libc/mingw/lib-common/msvcrt.def.in

```diff
@@ -1111,7 +1111,7 @@ _vprintf_l
 _vprintf_p
 _vprintf_p_l
 _vprintf_s_l
-F_NON_I386(_vscprintf) ; i386 _vscprintf replaced by emu
+_vscprintf
```

Comment:
Omit this change and then this PR is ready to be merged.

I tested this locally already.